doc/api/http.md

```diff
@@ -2195,6 +2198,7 @@ not abort the request or do anything besides add a `'timeout'` event.
 [`request.setTimeout()`]: #http_request_settimeout_timeout_callback
 [`request.socket.getPeerCertificate()`]: tls.html#tls_tlssocket_getpeercertificate_detailed
 [`request.socket`]: #http_request_socket
+[`request.destroy()`] #stream.html#stream_readable_destroy_error
```

Comment:
Nit: bottom references are usually sorted in ASCII order, so this line needs to be placed before the `` [`request.end()`]: `` line.